### ISSUE TITLE
ogg track times always 0:00 with latest imports

### ISSUE THREAD

**Issue by minusf** [beetbox/beets#3087]
### Problem

all new albums i imported since 31st october have `$track` set to `0:00`:

```
$ beet ls ./
Elixir - The Phobos Incident - Orpheus (Casino mix) - 0:00 - 01
Elixir - The Phobos Incident - Magic Orange (Blue Water mix) - 0:00 - 02
Elixir - The Phobos Incident - Isis - 0:00 - 03
Elixir - The Phobos Incident - Theme From Galaxian - 0:00 - 04
Elixir - The Phobos Incident - Prophetic Signs - 0:00 - 05
Elixir - The Phobos Incident - The Phobos Incident - 0:00 - 06
Elixir - The Phobos Incident - Deep Speed Diving - 0:00 - 07
Elixir - The Phobos Incident - Pegasus 51 (9mm mix) - 0:00 - 08
Elixir - The Phobos Incident - Dark Spheres of Yesterday - 0:00 - 09
Elixir - The Phobos Incident - 10 60 - 0:00 - 10
Elixir - The Phobos Incident - Alien Rainbow - 0:00 - 11
Elixir - The Phobos Incident - Deimos - 0:00 - 12
```

ogginfo output of first file:

```
Processing file "01-orpheus-casino-mix.ogg"...

New logical stream (#1, serial: 603ebc5f): type vorbis
Vorbis headers parsed for stream 1, information follows...
Version: 0
Vendor: Xiph.Org libVorbis I 20180316 (Now 100% fewer shells)
Channels: 2
Rate: 44100

Nominal bitrate: 192.000000 kb/s
Upper bitrate not set
Lower bitrate not set
User comments section follows...
	UPC=5410377900745
	ISRC=BE6F59700400
	ACOUSTID_FINGERPRINT=
	ACOUSTID_ID=
	ALBUM=The Phobos Incident
	ALBUM ARTIST=Elixir
	ALBUMARTIST=Elixir
	ALBUMARTIST_CREDIT=Elixir
	ALBUMARTISTSORT=Elixir
	MUSICBRAINZ_ALBUMCOMMENT=
	MUSICBRAINZ_ALBUMSTATUS=Official
	MUSICBRAINZ_ALBUMTYPE=album
	ARRANGER=
	ARTIST=Elixir
	ARTIST_CREDIT=Elixir
	ARTISTSORT=Elixir
	ASIN=
	BPM=0
	CATALOGNUMBER=Word D6
	DESCRIPTION=
	COMMENT=
	COMPILATION=0
	COMPOSER=
	COMPOSERSORT=
	RELEASECOUNTRY=GB
	DATE=1997
	YEAR=1997
	DISC=1
	DISCNUMBER=1
	DISCSUBTITLE=
	DISCTOTAL=1
	DISCC=1
	TOTALDISCS=1
	ENCODEDBY=
	ENCODER=
	GENRE=
	GROUPING=
	LABEL=Language
	PUBLISHER=Language
	LANGUAGE=eng
	LYRICIST=
	LYRICS=
	MUSICBRAINZ_ALBUMARTISTID=286a10c9-3436-43d7-af88-07157b960f75
	MUSICBRAINZ_ALBUMID=ba39ee99-e80a-4dc4-bc63-6232a8d9cb74
	MUSICBRAINZ_ARTISTID=286a10c9-3436-43d7-af88-07157b960f75
	MUSICBRAINZ_RELEASEGROUPID=ebe86ce5-8b25-4dce-96c9-9e13ee5129cd
	MUSICBRAINZ_RELEASETRACKID=ce0ecc77-86bb-45af-8fb0-715a220486f7
	MUSICBRAINZ_TRACKID=4c4f31fc-b107-4bab-808b-cb81b4249c14
	MEDIA=CD
	ORIGINALDATE=1997
	R128_ALBUM_GAIN=0
	R128_TRACK_GAIN=0
	SCRIPT=Latn
	TITLE=Orpheus (Casino mix)
	TRACK=1
	TRACKNUMBER=1
	TRACKTOTAL=12
	TRACKC=12
	TOTALTRACKS=12
Vorbis stream 1:
	Total data length: 5937112 bytes
	Playback length: 5m:27.199s
	Average bitrate: 145.161663 kb/s
Logical stream 1 ended
```

this was not an issue before 31st october.  ogg encoder version was the same.
python version was the same.  but macOS is new obviously, and there were
many `brew upgrades` since then...

### Setup

* OS: macos mojave 10.14.1
* Python version: 3.6.6
* beets version: 1.4.7
* Turning off plugins made problem go away (yes/no): 

My configuration (output of `beet config`) is:

```yaml
directory: /Volumes/256/sonoteque

format_item: $artist - $album - $title - $length - $track

paths:
    default: '%lower{$albumartist/$year-$album/$track-$title}'
    genre:anime: '%lower{ost-anime/$album-$year/$track-$title}'
    genre:game: '%lower{ost-game/$album-$year/$track-$title}'
    albumtype:soundtrack: '%lower{ost/$album-$year/$track-$title}'
    comp: '%lower{various/$album-$year/$track-$title}'

match:
    preferred:
        original_year: yes

import:
    from_scratch: yes

replace:
    \s*&\s*: -and-
    '[\\/]': ''
    '[\x00-\x1f]': '-'
    '[<>:,"\?\*\|!#\$]': ''
    \.\.: ''
    ^\.\.: ''
    \.$: ''
    \.\s: '-'
    \s\.: '-'
    ^\s+: ''
    \s+$: ''
    \s: '-'
    ^-: ''
    -$: ''
    -\.: '-'
    '[\[\]\(\)]': ''
    '''': ''
    -_-: '-'
asciify_paths: yes

plugins: edit fetchart ftintitle play
fetchart:
    sources: [filesystem]
    auto: yes
    minwidth: 0
    maxwidth: 0
    enforce_ratio: no
    cautious: no
    cover_names:
    - cover
    - front
    - art
    - album
    - folder
    google_key: REDACTED
    google_engine: 001442825323518660753:hrh5ch1gjzm
    fanarttv_key: REDACTED
    store_source: no
play:
    command: mpv
    raw: yes
    use_folders: no
    relative_to:
    warning_threshold: 100
ftintitle:
    auto: yes
    drop: no
    format: feat. {0}
edit:
    albumfields: album albumartist
    itemfields: track title artist album
    ignore_fields: id path
```

any ideas how to track this down?

**Comment by sampsyo** [comment]
Hmm… I guess I'm a little confused. You have this in your configuration:

```yams
format_item: $artist - $album - $title - $length - $track
```

which means that this output:

```
$ beet ls ./
Elixir - The Phobos Incident - Orpheus (Casino mix) - 0:00 - 01
```

implies that `$length` is 0:00, not `$track`. Is that what you meant?

If so, I'm not really sure what might have gone wrong… but maybe doing a `beet update` on one or two of the tracks would pick up the length from the file?

**Comment by sampsyo** [comment]
Wow; that's weird. Can you try it without the `from_scratch` config option (which clears tags and may be mistakenly clearing the length field too)?

```yaml
import:
    from_scratch: yes
```

**Comment by minusf** [comment]
good try, yes the `$length` is there now.

**Comment by sampsyo** [comment]
Got it! I think this was fixed in #2975? (I really need to cut a new release with this change...)
